### PR TITLE
Added section 4 (Jupyter)

### PR DESCRIPTION
**Pull request by frossie**

### CODE REVIEW

File: sec4-notebook.tex [hunk 1]
```diff
@@ -1,8 +1,57 @@
 \section{The Notebook Aspect}\label{notebook-aspect}
 
+The Science Platform offers in-house scientists and engineers as well as (in operations) general science users the opportunity to interactively explore the LSST dataset from a python notebook environment.
+
 \subsection{JupyterHub / JupyterLab service}\label{jupyterhub-jupyterlab-service}
 
+The notebook service is a scalable kubernetes deployment with JupyterHub spawning individual JupyterLab individual pods for each user. The basic architecture is outlined in \figref{fig:jlarch}. JupyterLab is a rich notebook architecture that offers terminal access and dashboarding in addition to the traditional Jupyter Notebook concept, and
```
Comment (ktlim) [inline sec4-notebook.tex]: Something missing from the end of this?  Capitalize "Kubernetes"?  "individual" is duplicated.

File: sec4-notebook.tex [hunk 2]
```diff
@@ -1,8 +1,57 @@
...
+
+A user of the JupyterLab sevice will hence be able to interact with LSST services in the same way they might have with python scripts in a shell environment. They can use our python interfaces to the DAX and the Butler for data access, the interface to the workflow system for batch processing, visualisation via bokeh, Firefly, matplotlib etc.
```
Comment (ktlim) [inline sec4-notebook.tex]: DAX -> Web APIs Aspect

File: sec4-notebook.tex [hunk 3]
```diff
@@ -1,8 +1,57 @@
...
+beta & Commissioning Team & LDM-503-9 & y\tabularnewline
+1.0 & Commissioning (ComCam) Capabilities & LDM-503-12 &
+n\tabularnewline
+2.0 & Science Verification Capabilities & LDM-503-14 & n\tabularnewline
+3.0 & General Science User Capabilities & LDM-503-16 & n\tabularnewline
+\bottomrule
+\end{longtable}
+\end{footnotesize}
+
+Note in particular that unlike the originally planned ``Level 3'' science user capabilities, the JL service is the primary interface of the LSST System Scientist to Commissioning. This introduces scope not required by the general science user, eg. timely access to data in the Engineering Facilities Database and the ability to rapidly deploy instances of the service on the Commissioning Cluster.
```
Comment (ktlim) [inline sec4-notebook.tex]: I'm not sure about the "unlike" here.  I think "in addition to" would be better.

File: sec4-notebook.tex [hunk 4]
```diff
@@ -1,8 +1,57 @@
...
+
+For analyses requiring significant data processing, a process for transitioning a trial-and-error notebook into a batch job involving execution over a large dataset will be provided. 
+
+For detailed architectural discussions of the JupyterLab development work-in progress, its interfaces to other components and the current schedule for major and internal milestones see \citeds{SQR-018}. As technical choices are baselined they will transition to this document.
```
Comment (ktlim) [inline sec4-notebook.tex]: "work-in-progress" or just "work in progress".

File: sec4-notebook.tex (hunk 1 above)
Comment (frossie) [reply sec4-notebook.tex]: It's not clear that kubernetes is capitalised (their web pages use both) and as it's a common noun in Greek it looks wrong to me to do so... Fixed rest.

File: sec4-notebook.tex (hunk 2 above)
Comment (frossie) [reply sec4-notebook.tex]: not sure what "aspect" is but replaced reference  to DAX with one to web API

File: sec4-notebook.tex (hunk 3 above)
Comment (frossie) [reply sec4-notebook.tex]: good call

**Pull request closed without merging**